src/mmrelay/config.py

```
@@ -333,6 +333,64 @@ def load_database_config_from_env():
+        for path in config_paths:
+            if os.path.isfile(path):
+                try:
+                    with open(path, "r", encoding="utf-8") as f:
+                        config = yaml.load(f, Loader=SafeLoader)
+                    if config:
+                        return is_e2ee_enabled(config)
+                except (yaml.YAMLError, PermissionError, OSError):
+                    continue  # Silently try the next path
+
+        # No valid config found
+        return False
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)